components/pkg-export-container/src/container.rs

```diff
@@ -144,18 +42,21 @@ pub struct ContainerImage {
     tags:    Vec<String>,
     /// The base workdir which hosts the root file system.
     workdir: PathBuf,
-}
-
-impl Identified for ContainerImage {
-    fn name(&self) -> String { self.name.clone() }
 
-    fn tags(&self) -> Vec<String> { self.tags.clone() }
+    /// All the identifiers for this image (in {name}:{tag} format)
+    expanded_identifiers: Vec<String>,
 }
 
 impl ContainerImage {
     // TODO (CM): temporary; we shouldn't use this at all
     pub fn workdir(&self) -> &Path { self.workdir.as_path() }
 
+    pub fn expanded_identifiers(&self) -> &Vec<String> { &self.expanded_identifiers }
```

Comment:
Can we make the return type `&[String]`?

Reply:
![](https://media.giphy.com/media/l0ErOholJjSmFlMFG/giphy.gif)

Reply:
Addressed in https://github.com/habitat-sh/habitat/pull/7682/commits/decc37827a8b30249603803fcba5f9f7d5922bbe